Assets/Events/Adventurers Join.json

```diff
@@ -347,7 +347,7 @@
 	{
 		"name": "boring",
 		"headline": "Just a Normal Party",
-		"article": "A group of bland looking adventurers walk into town today. When questioned, they have no quirks, no strange backstory. They're just, here?\n\n<i>\"Look, we can't ALL be funny and unique, can we?\"</i>, the party leader says, in an incredibly monotone voice.",
+		"article": "A group of bland looking adventurers walk into town today. When studied, they seem to have no quirks, and no strange backstory. They're just, here?\n\n<i>\"Look, we can't ALL be funny and unique, can we?\"</i>, the party leader says, in an incredibly bland and forgettable voice.",
```

Comment:
I think this needs a third in the list for the right flow i.e. 
Alternatively we should have a third thing after backstory, maybe "no traumatic backstory, no quirks, and no grand worldending quest... They're just, here?"

Maybe something a bit funnier if you have ideas

Reply:
Third definitely makes it better. I've left it at that for now.